This patch, sent by Janos Farkas <chexum@shadow.banki.hu>, is needed
for automake-1.0 to generate correct Makefiles for this package.
Users don't need this, but developers do.  Also make sure that you
have GNU autoconf-2.10 or later.  Don't ask me why :).  --marekm

*** automake.in.orig	Sat May 25 18:36:05 1996
--- automake.in	Wed Jul  3 02:49:47 1996
***************
*** 983,997 ****
      # We generate the manpage install code by hand to avoid the use of
      # basename in the generated Makefile.
      local (@mans) = split (' ', $contents{'man_MANS'});
!     local (%sections, %inames, %secmap, %fullsecmap);
      foreach (@mans)
      {
  	# FIXME: statement without effect:
  	/^(.*)\.([0-9])([a-z]*)$/;
  	$sections{$2} = 1;
! 	$inames{$1} = $_;
! 	$secmap{$1} = $2;
! 	$fullsecmap{$1} = $2 . $3;
      }
  
      # We don't really need this, but we use it in case we ever want to
--- 983,1000 ----
      # We generate the manpage install code by hand to avoid the use of
      # basename in the generated Makefile.
      local (@mans) = split (' ', $contents{'man_MANS'});
!     local (%sections, %inames, %mbases, %secmap, %fullsecmap);
!     $i = 1;
      foreach (@mans)
      {
  	# FIXME: statement without effect:
  	/^(.*)\.([0-9])([a-z]*)$/;
  	$sections{$2} = 1;
! 	$inames{$i} = $_;
! 	$mbases{$i} = $1;
! 	$secmap{$i} = $2;
! 	$fullsecmap{$i} = $2 . $3;
! 	$i++;
      }
  
      # We don't really need this, but we use it in case we ever want to
***************
*** 1016,1022 ****
  	s/\@SECTION\@/$secmap{$key}/g;
  	s/\@MAN\@/$inames{$key}/g;
  	s/\@FULLSECT\@/$fullsecmap{$key}/g;
! 	s/\@MANBASE\@/$key/g;
  	$output_rules .= $_;
      }
      $output_rules .= "\n";
--- 1019,1025 ----
  	s/\@SECTION\@/$secmap{$key}/g;
  	s/\@MAN\@/$inames{$key}/g;
  	s/\@FULLSECT\@/$fullsecmap{$key}/g;
! 	s/\@MANBASE\@/$mbases{$key}/g;
  	$output_rules .= $_;
      }
      $output_rules .= "\n";
***************
*** 1028,1034 ****
  	s/\@SECTION\@/$secmap{$key}/g;
  	s/\@MAN\@/$inames{$key}/g;
  	s/\@FULLSECT\@/$fullsecmap{$key}/g;
! 	s/\@MANBASE\@/$key/g;
  	$output_rules .= $_;
      }
      $output_rules .= "\n";
--- 1031,1037 ----
  	s/\@SECTION\@/$secmap{$key}/g;
  	s/\@MAN\@/$inames{$key}/g;
  	s/\@FULLSECT\@/$fullsecmap{$key}/g;
! 	s/\@MANBASE\@/$mbases{$key}/g;
  	$output_rules .= $_;
      }
      $output_rules .= "\n";
